src/Activity/TrainWithExample/ChooseExample.js

```diff
@@ -36,9 +43,71 @@ const styles = theme => ({
   title: {
     display: "flex",
     alignItems: "center"
+  },
+  adverb: {
+    color: "red"
   }
 });
 
+
+
+const images = (activityChosen) => {
+  return {
+    mammals: mammalsData,
+    parallelograms: parallelogramData,
+    adverbs: adverbsData
+  }[activityChosen];
+}
```

Comment:
No need to have a function here. I suggest to do: 
```
const topicData {
￼     mammals: mammalsData,
￼     parallelograms: parallelogramData,
￼     adverbs: adverbsData
￼ }
```
and below in the code instead or `images(activitChosen)` do `topicData[activityChosen]`